moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -94,12 +94,33 @@ const validationSchema = yup.object().shape({
 
 const validator = (value) => yupValidator(value, validationSchema);
 
-// Lookup array to convert project note types to a human readable interpretation
-// The zeroth item in the list is intentionally blank; the notes are 1-indexed.
-const projectNoteTypes = ["", "Internal Note", "Status Update"];
+// reshape the array of note types into an object with key slug, value id
+export const useNoteTypeObject = (noteTypes) =>
+  useMemo(
+    () =>
+      noteTypes.reduce(
+        (obj, item) =>
+          Object.assign(obj, {
+            [item.slug]: item.id,
+          }),
+        {}
+      ),
+    [noteTypes]
+  );
 
-export const STATUS_UPDATE_TYPE_ID = 2;
-export const INTERNAL_NOTE_TYPE_ID = 1;
+const useFilterNotes = (notes, filterNoteType) =>
+  useMemo(() => {
+    if (filterNoteType === 0) {
```

Comment:
i didnt know what 0 represented here, could be helpful to add a comment w context that this is when the notes are not being filtered by type. or maybe this state var could just be null when theres no note type filter being applied